src/main/java/edu/harvard/iq/dataverse/usage/StringUtil.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package cn.edu.pku.lib.util;
```

Comment:
I changed the package name to without noticing it since I used the maven to build it successful.  I  am surprised too!